spec/ios_download_strings_files_from_glotpress_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+require 'tmpdir'
+
+Locale = Struct.new(:name)
+
+describe Fastlane::Actions::IosDownloadStringsFilesFromGlotPressAction do
+  let(:test_data_dir) { File.join(File.dirname(__FILE__), 'test-data', 'translations', 'ios_generate_strings_file_from_code') }
+
+  describe 'downloading export files from GlotPress' do
+    it 'downloads all the locales into the expected directories'
+    it 'uses the proper filters when exporting the files from GlotPress'
+    it 'uses a custom table name for the `.strings` files if provided'
+  end
+
+  describe 'error handling' do
+    it 'shows an error if an invalid locale is provided (404)'
+    it 'shows an error if the file cannot be written in the destination'
+    it 'reports if a downloaded file is invalid by default'
+    it 'does not report invalid downloaded files if skip_file_validation:true'
+  end
```

Comment:
PR is still Draft, I plan to implement these soon… though having the helper from #330 merged first to then use it to implement those would help 🙂